core/include/webview/detail/backends/cocoa_webkit.hh

```diff
@@ -616,6 +557,11 @@ private:
     add_init_script("function(message) {\n\
```

Comment:
From here `bind` is indirectly executed in the class initialisation flow. This may have been spawning reciprocal executions of the `set_default_size` lamda, but this should now be reduced to one.

Reply:
I put this comment under the wrong backend, relates to win32_edge.
See https://github.com/webview/webview/pull/1249/files#r2009105779